### ISSUE TITLE
Feature request: Add additional color stops

### ISSUE THREAD

**Issue by mortocks** [SimeonGriggs/tailwind-css-palette-generator#18]
Would be amazing to be able to add additional color stops above and below e.g adding 25 or 950. Useful for working with legacy color pallets or non-tailwind based systems.

(love the tool btw - thanks!)

**Comment by SimeonGriggs** [comment]
I do have a branch sitting around to add just that, I never finished it 🙃 

Maybe one day...

**Comment by vctormb** [comment]
yeah it would be great! I'm using a color palette of 12 steps. Amazing job on this tool btw

**Comment by SimeonGriggs** [comment]
950's now in there (in preparation for the next version of Tailwind) but configurable stops would be neat too!

**Comment by gerdadesign** [comment]
+1 for request to add more stops as needed! 

Ex. + in between 100 and 200 would add 150. + between 400 and 500 would add 450. Would be nice to have an option to add these for some instances where we need more colors (in my case a neutrals scale).
